app/connectors/ReportDeadlinesConnector.scala

```diff
@@ -36,7 +36,7 @@ class ReportDeadlinesConnector @Inject()(val http: HttpClient,
                                           ) extends RawResponseReads {
 
   val getReportDeadlinesUrl: String => String =
-    nino => s"${appConfig.desUrl}/enterprise/obligation-data/nino/$nino/ITSA"
+    selfEmploymentId => s"${appConfig.desUrl}/enterprise/obligation-data/mtdbis/$selfEmploymentId/ITSA?status=O"
```

Comment:
This won't be called `selfEmploymentId` if the income source is property. Instead it has a generic name of `incomeSourceId`. So can this be changed to that please :)